app/lib/core/BaseModel.php

```diff
@@ -3725,11 +3725,25 @@ public function _processMedia($ps_field, $pa_options=null) {
 					}
 				}
 
+//smc: overriding temp file and path
+/*
+if (isset($_REQUEST['jfu_media_file_name']))
+    $jfu_media_file_name = $_REQUEST['jfu_media_file_name'];
+else
+    $jfu_media_file_name = 'nothing';
+    
+error_log( print_r( 'smc jfu_media_file_name='.$jfu_media_file_name, true ) ); 
+                
+$this->_SET_FILES[$ps_field]['tmp_name'] = "/nfs/images/tmp/jfu/".$jfu_media_file_name; 
+$this->_SET_FILES[$ps_field]['original_filename'] = $jfu_media_file_name;                
+*/                
 				// allow adding zip and (gzipped) tape archives
 				$vb_is_archive = false;
 				$vs_archive = null;
 				$vs_original_filename = $this->_SET_FILES[$ps_field]['original_filename'];
 				$vs_original_tmpname = $this->_SET_FILES[$ps_field]['tmp_name'];
+error_log(print_r('smc in BaseModel $vs_original_filename='.$vs_original_filename.'------', true));
```

Comment:
Are the various error_log calls still needed/helpful? Maybe keep in for dev testing, remove for move to production?